dot/peerset/peerstate.go

```diff
@@ -354,16 +359,13 @@ func (ps *PeersState) forgetPeer(set int, peerID peer.ID) error {
 }
 
 // tryOutgoing tries to set the peer as connected as an outgoing connection.
-// If there are enough slots available, switches the node to Connected and returns nil error. If
-// the slots are full, the node stays "not connected" and we return error.
+// If there are enough slots available, switches the node to Connected and returns
+// nil error. If the slots are full, the node stays "not connected" and we return error.
```

Comment:
out of scope, but whilst we're at it

```suggestion
// a nil error. If the slots are full, the node stays "not connected" and we return an error.
```

Reply:
changed